leshan-core/src/main/java/org/eclipse/leshan/util/Key.java

```diff
@@ -0,0 +1,39 @@
+package org.eclipse.leshan.util;
```

Comment:
I put it here because this is not californium dependent and this can be reused for anyone we want to use byte array as key for a map.